src/components/AppNavigation/CalendarList.vue

```diff
@@ -99,3 +196,20 @@ export default {
 	},
 }
 </script>
+
+<style scoped>
+:deep(.list-item) {
+	padding: 4px 8px 4px 8px;
+	margin-bottom: 0 !important;
```

Comment:
deep and !important are often a code smell. would it possible to achieve this without? e.g. by moving the styles into the child or upstreaming conditional styles via a prop in nc/vue?